.github/actions/stale_issues/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Filter stale issues
+inputs:
+  token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
+  period:
+    description: "The stale period in days"
+    required: false
+    type: number
+    default: 14
+  repo:
+    description: "The target repository"
+    required: false
+    type: string
+    default: "${{ github.repository_owner }}/${{ github.event.repository.name }}"
+outputs:
+  result:
+    description: "The result of the query"
+    value: ${{ steps.found_issues.outputs.result }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get period date
+      id: date
+      shell: bash
+      run: |
+        period_date=$(date --date "${{ inputs.period }} days ago" --utc +"%Y-%m-%dT%H:%M:%SZ")
+        echo "period=$period_date" >> $GITHUB_OUTPUT
+    - name: Get inactive open issues
+      id: stale_issues
+      if: ${{ success() }}
+      uses: ethereum/solidity/.github/actions/query_issues@develop
```

Comment:
Is there some variable we could use for default branch and repo name instead of hard-coding them?

Actually, can this be a relative path like in the other .yml file?

Reply:
I did in this way to be easy to call the actions from another repository without need to checkout in the action as well. Because this action (stale_issues) calls another (query_issues).

As far as I know, the relative path only works if you do the checkout to the repository that contains the actions/workflows that you want to reuse.